canonical-websites/www.ubuntu.com
# Add Launchpad terms of service

This is lifted directly from https://help.launchpad.net/Legal, except for the privacy policy section; that will become a Launchpad-specific privacy notice under /legal/dataprivacy once legal have finished
drafting it.

The text has been through a [round of review](https://docs.google.com/document/d/1b_MkB4ejhOQ1IWtXIOBDa38G8pw2NhJFF_TfTlnzSqE/edit) by legal, and they've requested that it be moved to www.ubuntu.com.

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the relevant part of the site locally in your web browser at: [http://0.0.0.0:8001/legal/terms-and-policies](http://0.0.0.0:8001/legal/terms-and-policies) and confirm that the Launchpad terms section looks reasonable
